### ISSUE TITLE
Blocked Images Counter

### ISSUE THREAD

**Issue by Woctor-Dho** [nsfw-filter/nsfw-filter#78]
#### Is your feature request related to a problem? Please describe.
There is currently little feedback to the user as to how many/where modifications are made to the page. 

#### Describe the solution you'd like
A counter to show the number of blocked images. 
Example: uBlock Origin shows the number of items on the page blocked.
![image](https://user-images.githubusercontent.com/25572322/92135096-dedebb80-edf9-11ea-875d-4870185037cf.png)

#### Describe alternatives you've considered
\-

#### Additional context
Would work well in combination with the watermark feature proposed in #2

**Comment by YegorZaremba** [comment]
Hi @Woctor-Dho great idea, thanks for contributing
We already have a variable for this https://github.com/nsfw-filter/nsfw-filter/blob/master/src/content/Filter/ImageFilter.ts#L37
On hold until `popup` design is done

**Comment by YegorZaremba** [comment]
Agree UX +1, we show the number of blocked images for the website in the active tab, because we don't store this data anywhere

But if talk about the total amount of blocked images - popup is the best place(and we store this data)

**Comment by navendu-pottekkat** [comment]
I think the number of blocked images in the active pages is enough right? What do you think @Woctor-Dho ?

**Comment by Woctor-Dho** [comment]
@navendu-pottekkat I agree, the number for the active page would be ideal. Although the all-time total is a cool stat, current page is more useful (as a badge) for quick, at-a-glance feedback to the user. This would also be more intuitive as showing current page stats is a common behavior among many extensions.

**Comment by navendu-pottekkat** [comment]
Yeah, so the best way to do would be to add a badge that shows images blocked in the active tab and we can add a popup with the total lifetime block as some stats and add some settings for the user(watermark etc) What do you think? @Woctor-Dho @YegorZaremba

**Comment by navendu-pottekkat** [comment]
@all-contributors please add @Woctor-Dho for ideas
